packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1383,6 +1498,18 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
 export const compareSequenceIntervalEnds = (a: SequenceInterval, b: SequenceInterval): number =>
 	compareReferencePositions(a.end, b.end);
 
+export const compareSequenceIntervalStarts = (a: SequenceInterval, b: SequenceInterval): number =>
+	compareReferencePositions(a.start, b.start);
+
+export const createSequenceIntervalHelper = (): IIntervalHelpers<SequenceInterval> => {
```

Comment:
seems like you could just export a `sequenceIntervalHelpers` rather than a creation function.